File: bddtests/src/main/kotlin/mocking/tpp/models/RequestSystmOnlineMessagesReply.kt
package mocking.tpp.models

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlAttribute
import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement(name = "RequestSystmOnlineMessagesReply")
@XmlAccessorType(XmlAccessType.FIELD)
data class RequestSystmOnlineMessagesReply (
        @XmlAttribute var patientId: String = "default patientId",
        @XmlAttribute var onlineUserId: String = "default onlineUserId",
        @XmlAttribute var uuid: String = "default uuid",
        @XmlAttribute(name = "BookAppointments") var bookAppointments: String? = "default BookAppointments"
)

File: backendworker/NHSOnline.Backend.GpSystems/Suppliers/Tpp/Models/RequestSystmOnlineMessagesReply.cs
using System;
using System.Xml.Serialization;

namespace NHSOnline.Backend.GpSystems.Suppliers.Tpp.Models
{
    [Serializable]
    public class RequestSystmOnlineMessagesReply
    {
        [XmlAttribute("patientId")]
        public string PatientId { get; set; }

        [XmlAttribute("onlineUserId")]
        public string OnlineUserId { get; set; }

        [XmlAttribute("uuid")]
        public string UuId { get; set; }

        [XmlAttribute("BookAppointments")]
        public string BookAppointments { get; set; }

        [XmlAttribute("RequestMedicationConfirmation")]
        public string RequestMedicationConfirmation { get; set; }

        [XmlAttribute("Medication")]
        public string Medication { get; set; }
    }
}